src/openkit-shared/Protocol/HTTPClient.cs

```diff
@@ -116,13 +116,13 @@ protected Response SendRequest(RequestType requestType, string url, string clien
             {
                 if (logger.IsDebugEnabled)
                 {
-                    logger.Debug("HTTP " + requestType.RequestName + " Request: " + url);
+                    logger.Debug("HTTPClient HTTP " + requestType.RequestName + " Request: " + url);
```

Comment:
I'd prefer to have a more "general" way of adding the class name (e.g. via reflection).